packages/tools/src/utilities/segmentation/createLabelmapVolumeForViewport.ts

```diff
@@ -44,7 +44,7 @@ export default async function createLabelmapVolumeForViewport(input: {
   }
 
   const { viewport } = enabledElement;
-  if (!(viewport instanceof VolumeViewport)) {
+  if (!(viewport instanceof BaseVolumeViewport)) {
```

Comment:
We currently don't have Seg support (or not tried it maybe) on the 3D viewport, maybe rever this?

Reply:
You're right. Will do.